sale_exception/README.rst

```diff
@@ -37,6 +37,42 @@ found in Odoo, wrong line format etc.)
 .. contents::
    :local:
 
+Configuration
+=============
+
+To configure this module you need to:
+
+#. Go to *Settings > User & Companies > Users*.
+#. Edit your user.
+#. Check "A warning can be set on a product or a customer (Sale)" group.
```

Comment:
This is only if you want to use such exception, but not for others, so you should specify it.